### DISCUSSION TITLE
某些专有名词不能被正确翻译，能否对翻译结果的某个词的全部替换

### DISCUSSION THREAD

**Discussion by srsng** [immersive-translate/immersive-translate#465] [Ideas]
### 新功能 | New Feature

1.对某个域名的某些词（展示的译文中）全部替换成用户指定的。
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.python被翻译成蟒蛇，rust翻译成铁锈等等并不让人喜欢
2.
3.

**Comment by theowenyoung** [comment]
这个属于术语表的功能，之前想做的很复杂，但是重新想了想，是不是也可以做的很简单，就是指定词语不翻译，列表中的词通通保持原样是不是就可以了？

**Reply by srsng** [reply]
感谢回复！
我认为可以为用户创建一个表

> {
----域名:   
----{  
--------翻译后的某个词_1 : 用户指定的译词(或原词) ,
--------...  
----}  
----...
} 

这样对每个域名管理，也期待更好的管理方式。

**Reply by theowenyoung** [reply]
正常来讲，还要考虑语言， en-zh, ja->zh 每个语言对应的术语也不一样...

**Reply by srsng** [reply]
这些由用户解决，用户觉得某个词不对，可以把这个词添加到列表中，然后交给程序替换。当然，列表中也要提及语言与翻译方向

**Reply by theowenyoung** [reply]
对，可以是可以，我只是觉得这会是一个非常复杂的旁路逻辑，如果用简单的方式可以解决80%的问题的话，那可能简单的处理会更适合大多数人。使用起来没那么复杂。
